CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ in the naming of release branches.
 ## [Unreleased]
 
 ### Added
+- New `calculatePoolDistr'` function which is similar to `calculatePoolDistr` but has a new
+  filter argument to only include the stake pool ids (stake pool key hashes) that are needed.
```

Comment:
we've been adding in the PR number here to, which is quite awkward at the moment (since you have to put up the PR, see what number you get, then push). the hope is to get some hook that'll do this for us. apologies for forgetting to tell you this.

Reply:
Done!